Assets/ModScripts/Submodules/UnscrewMaze.cs

```diff
@@ -0,0 +1,145 @@
+﻿using System;
+using System.Linq;
+using UnityEngine;
+
+public class UnscrewMaze : Puzzle
+{
+
+    readonly string[] maze = { "1", "13", "23", "12", "13", "23",
+                               "12", "3", "02", "0", "2", "02",
+                               "012", "3", "012", "13", "013", "03",
+                               "01", "123", "013", "13", "23", "2",
+                               "23", "02", "1", "123", "03", "02", 
```

Comment:
```suggestion
                               "2", "02", "1", "123", "03", "02", 
```
The 3 makes you leave the maze